Scripts/DatabankLib/settings/engines.py

```diff
@@ -896,3 +495,71 @@
     "CHARMM": charmm_dict,
     "OPENMM": openmm_dict,
 }
+
+
+def get_3major_fnames(system: dict, allowStructure=False, joinPath=None):
```

Comment:
The minor thing is the name of the function. What is wrong with naming it by three names?

`def get_struct_top_traj_fnames() -> tuple[str, str, str]:`

In my opinion this does not create additional confusion and does not require the developer to think of what those 3 major fnames are.

Reply:
I decided to use this `tuple[Optional[str], Optional[str], Optional[str]]` syntax instead. It works with py<310

Reply:
done